web/db/characters.ts

```diff
@@ -32,6 +49,7 @@ export type Character = {
   type: "Character";
   name: string;
   wiki: string;
+  talentMaterialWeekly: TalentMaterial[] | null; // Ayaka and Xiao does not have weeklies yet.
```

Comment:
Is there a particular reason you went with `TalentMaterial[] | null`? I think an empty array would be good enough to indicate the lack of weekly material.

Reply:
My bad, didn't cross my mind yesterday. Sorry! Fixed!